ui/src/components/logs/ContainerLogsView.js

```diff
@@ -175,12 +192,29 @@ export const ContainerLogsView = ({
         </span>
       </EuiTitle>
       <EuiSpacer size="s" />
+      {
+        imagebuilderStackdriverUrl && (
+          <EuiCard textAlign="left" title="">
+          <EuiDescriptionList 
+          style={{ marginBottom: "8px"}} 
+          titleProps={{ style: { width: "30%", fontSize: '14px'} }}
+          type="column" 
+          >
+          <EuiDescriptionListTitle style={{ width: "30%", fontSize: '14px'}} >Stackdriver Logs</EuiDescriptionListTitle>
+          <EuiDescriptionListDescription>
+            <EuiLink href={imagebuilderStackdriverUrl} target="_blank" external>Image Builder</EuiLink>
+          </EuiDescriptionListDescription>
+          </EuiDescriptionList>
+        </EuiCard>
+        )
+      }
+      <EuiSpacer size="s" />
```

Comment:
This EuiSpacer should be inside the conditional block above:

```
{
        imagebuilderStackdriverUrl && (
          <>
            <EuiCard>...</EuiCard>
            <EuiSpacer size="s" />
          </>
        )
}
```

Reply:
i knew but didnt want to add a `<Fragment>` for it so i left it like this 🤣 , added the fragment and spacer anyway, thanks!